src/Application/PageApprovalAuthorizer.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\PageApprovals\Application;
+
+use MediaWiki\Page\PageIdentity;
+
+interface PageApprovalAuthorizer {
```

Comment:
`ApprovalAuthorizer` should be sufficient

Reply:
https://github.com/ProfessionalWiki/PageApprovals/pull/10